docs/products/kafka/kafka-mirrormaker/howto/integrate-external-kafka-cluster.rst

```diff
@@ -20,4 +20,108 @@ You can define an external Apache Kafka® service integration endpoint using the
+
+For external source the read permission is required to the mirrored topics and on external target the write permission is required to the mirrored topics.
+For the internal topics the read and write permissions are required for the connecting user.
+
+The set of topics created by Apache Kafka® MirrorMaker2 to source cluster is the following.
+
+.. list-table::
+  :header-rows: 1
+
+  * - Topic
+    - | Replication factor
+      | (default value)
+    - | Partitions
+      | (default value)
+    - | Cleanup policy
+      | (required policy)
+  * - ``__consumer_offsets``
+    - 3
+    - 50
+    - compact
+  * - ``heartbeats``
+    - 3
+    - 1
+    - compact
+  * - ``mm2-configs.<target cluster alias>.internal``
+    - 3
+    - 1
+    - compact
+  * - | ``mm2-offset-syncs.<target cluster alias>.internal``
+      | see `Offsets sync topic location`_
+    - 3
+    - 1
+    - compact
+  * - ``mm2-offsets.<target cluster alias>.internal``
+    - 3
+    - 25
+    - compact
+  * - ``mm2-status.<target cluster alias>.internal``
+    - 3
+    - 5
+    - compact
+
+
+The set of topics created by Apache Kafka® MirrorMaker2 to target cluster is the following:
+
+.. list-table::
+  :header-rows: 1
+
+  * - Topic
+    - | Replication factor
+      | (default value)
+    - | Partitions
+      | (default value)
+    - | Cleanup policy
+      | (required policy)
+  * - ``__consumer_offsets``
+    - 3
+    - 50
+    - compact
+  * - ``heartbeats``
+    - 3
+    - 1
+    - compact
+  * - ``<source cluster alias>.checkpoints.internal``
+    - 3
+    - 1
+    - compact
+  * - ``<source cluster alias>.heartbeats``
+    - 3
+    - 1
+    - compact
+  * - ``mm2-configs.<source cluster alias>.internal``
+    - 3
+    - 1
+    - compact
+  * - ``mm2-offsets.<source cluster alias>.internal``
+    - 3
+    - 25
+    - compact
+  * - ``mm2-status.<source cluster alias>.internal``
+    - 3
+    - 5
+    - compact
+
+
+If using heartbeat emitting to source cluster, configuration is called ``emit_backward_heartbeats_enabled``, the read and write is required to ``mm2-offsets.<target cluster alias>.internal`` topic at the external source cluster. When disabled the topic and permissions are not required at the source cluster.
+Similarly if using heartbeat emitting to target cluster, configuration is called ``emit_heartbeats_enabled``, the read and write is required to ``mm2-offsets.<source cluster alias>.internal``. When disabled the topic and permissions are not required at the target cluster.
+
+
+.. note::
+   Summarizing the requirements for external Kafka cluster integration, configure the ACLs for both the source and target cluster such that the Apache Kafka® MirrorMaker 2 service can describe and create topics, as well as produce and consume messages.
```

Comment:
```suggestion
   To summarize, for external Kafka cluster integration, configure the ACLs for both the source and target clusters. This will enable Apache Kafka® MirrorMaker 2 to describe and create topics, as well as produce and consume messages.
```

Reply:
Done.